addons/mrp/models/stock_move.py

```diff
@@ -22,6 +23,10 @@ def _compute_picking_type_id(self):
             line_to_remove |= line
         return super(StockMoveLine, self - line_to_remove)._compute_picking_type_id()
 
+    def _search_picking_type_id(self, operator, value):
```

Comment:
`def _get_domain_picking_type_id(self, operator, value):`
it return a domain.

Reply:
It's a search method used for computed field 
https://www.odoo.com/documentation/14.0/developer/misc/other/guidelines.html#symbols-and-conventions
![image](https://user-images.githubusercontent.com/12071695/132308136-6e0a0078-2104-4ccb-aa39-68cef610848c.png)
I had to override it in mrp but it's defined in stock